pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -41,6 +41,8 @@ type ChaosEngineSpec struct {
 	JobCleanUpPolicy string `json:"jobCleanUpPolicy,omitempty"`
 	//AuxiliaryAppInfo contains details of dependent applications (infra chaos)
 	AuxiliaryAppInfo string `json:"auxiliaryAppInfo,omitempty"`
+	//Tracking enables or dsables Google-Analytics event provisioning
+	Tracking bool `json:"tracking"`
```

Comment:
This is also not required, As we are using chaos-operator yaml to manipulate the tracking status, This is for chaosEngine

Reply:
Okay Done !